tools/wptrunner/wptrunner/tests/test_formatters.py

```diff
@@ -0,0 +1,61 @@
+import json
+import sys
+import time
+from os.path import dirname, join
+from StringIO import StringIO
+
+from mozlog import handlers, structuredlog
+
+sys.path.insert(0, join(dirname(__file__), "..", ".."))
+
+from wptrunner.formatters import WptreportFormatter
+
+
+def test_wptreport_runtime(capfd):
+    # setup the logger
+    output = StringIO()
+    logger = structuredlog.StructuredLogger("test_a")
+    logger.add_handler(handlers.StreamHandler(output, WptreportFormatter()))
+
+    # output a bunch of stuff
+    logger.suite_start(["test-id-1"], run_info={})
+    logger.test_start("test-id-1")
+    time.sleep(0.125)
+    logger.test_end("test-id-1", "PASS")
+    logger.suite_end()
+
+    # check nothing got output to stdout/stderr
+    # (note that mozlog outputs exceptions during handling to stderr!)
+    captured = capfd.readouterr()
+    assert captured.out == ""
+    assert captured.err == ""
+
+    # check the actual output of the formatter
+    output.seek(0)
+    output_obj = json.load(output)
+    assert output_obj["results"][0]["runtime"] >= 125
```

Comment:
Sos, is this going to end up intermittent? :)

Reply:
It shouldn't unless `time.sleep` sleeps for less than 125ms at least. Though maybe we'll run into timer resolution issues, actually. Ergh.

Reply:
Can we just try landing it, though, and see if it actually flaky? I'd hope timer resolution issues would mean we'll just have a longer runtime? (Also `runtime` or `run_time`?)

Reply:
`duration` if you ask me, but I really don't mind.

I'm happy to land this I guess, but I would be more nervous if we were running tests on Windows XP or something. I would also just be happy to check that the value is a number > 0.

Reply:
It would appear the Windows APIs mostly make no guarantee as to rounding, which is fun. Requiring 62ms should be fine, though, AFAICT.